### PR TITLE
selinux-policy: Update to refpolicy 2.20221101.

### PR DESCRIPTION
**Pull request by pebenito**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Update to SELinux Reference policy 2.20221101.

###### Change Log 
- Update to SELinux Reference policy 2.20221101.
- Fix iscsid access to initiatorname.iscsi.
- Add file context for /etc/multipath/*
- Handle /media and /srv symlinks.
- Add cloud-init support for installing RPMs and setting passwords.

###### Does this affect the toolchain? 
**NO**

###### Test Methodology
- Pipeline build id: 325205
- Local testing
- Static analysis comparison to prior version.

### CODE REVIEW

File: SPECS/selinux-policy/0026-unconfined-Add-remaining-watch_-permissions.patch [hunk 1]
```diff
@@ -0,0 +1,116 @@
+From 3aedf6e07370b3845e9a7ca82ce6b59aad81fc1e Mon Sep 17 00:00:00 2001
```
Comment (dburgener) [inline SPECS/selinux-policy/0026-unconfined-Add-remaining-watch_-permissions.patch:1]: Can this go upstream?

Comment (pebenito) [reply SPECS/selinux-policy/0026-unconfined-Add-remaining-watch_-permissions.patch:1]: Yes.

File: SPECS/selinux-policy/0004-systemd-Misc-updates.patch [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+diff --git a/policy/modules/system/init.te b/policy/modules/system/init.te
+index 97a75cf86..d8b3e9fa3 100644
+--- a/policy/modules/system/init.te
++++ b/policy/modules/system/init.te
+@@ -495,6 +495,7 @@ ifdef(`init_systemd',`
+ 	storage_getattr_removable_dev(init_t)
+ 
+ 	term_relabel_pty_dirs(init_t)
++	term_watch_reads_unallocated_ttys(init_t)
```
[the remark this answers is not in the document]
Comment (pebenito) [reply SPECS/selinux-policy/0004-systemd-Misc-updates.patch:19]: You're correct, no longer needed.

File: SPECS/selinux-policy/0001-cronyd-Add-dac_read_search.patch [hunk 3]
```diff
@@ -1,28 +1,26 @@
...
 Date: Fri, 20 Aug 2021 18:11:23 +0000
-Subject: [PATCH 2/8] cronyd: Add dac_read_search.
+Subject: [PATCH 01/36] cronyd: Add dac_read_search.
 
 Signed-off-by: Chris PeBenito <Christopher.PeBenito@microsoft.com>
 ---
  policy/modules/services/chronyd.te | 2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)
 
-MSFT_TAG: pending
```
[the remark this answers is not in the document]
Comment (pebenito) [reply SPECS/selinux-policy/0001-cronyd-Add-dac_read_search.patch:11]: I forgot to add them back after regenerating the patches.

File: SPECS/selinux-policy/0007-unconfined-Manage-own-fds.patch [hunk 4]
```diff
@@ -1,7 +1,7 @@
-From d8884f88c40667c4b1118044b8e47f58bc54b92e Mon Sep 17 00:00:00 2001
+From a498730f40bcb5a1df2410aeaa05a5fd43c2ced3 Mon Sep 17 00:00:00 2001
 From: Chris PeBenito <Christopher.PeBenito@microsoft.com>
 Date: Wed, 15 Jun 2022 14:43:12 +0000
-Subject: [PATCH 39/39] unconfined: Manage own fds.
+Subject: [PATCH 07/36] unconfined: Manage own fds.
```
[the remark this answers is not in the document]
Comment (pebenito) [reply SPECS/selinux-policy/0007-unconfined-Manage-own-fds.patch:4]: No reason.

**Review by dburgener — APPROVED** [review]

**Review by jslobodzian — APPROVED** [review]
Approved with comments.

File: SPECS/selinux-policy/0014-sysnetwork-ifconfig-searches-debugfs.patch [hunk 5]
```diff
@@ -1,18 +1,20 @@
...
 Date: Mon, 29 Aug 2022 20:07:54 +0000
-Subject: [PATCH 48/48] sysnetwork: ifconfig searches debugfs.
+Subject: [PATCH 14/35] sysnetwork: ifconfig searches debugfs.
 
 Signed-off-by: Chris PeBenito <Christopher.PeBenito@microsoft.com>
 ---
  policy/modules/system/sysnetwork.te | 1 +
  1 file changed, 1 insertion(+)
 
+MSFT_TAG: pending
```
[the remark this answers is not in the document]
Comment (pebenito) [reply SPECS/selinux-policy/0014-sysnetwork-ifconfig-searches-debugfs.patch:11]: I will be upstreaming the patch.

File: SPECS/selinux-policy/0033-domain-Unconfined-can-transition-to-other-domains.patch [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+From: Chris PeBenito <Christopher.PeBenito@microsoft.com>
+Date: Wed, 22 Feb 2023 19:45:50 +0000
+Subject: [PATCH 33/35] domain: Unconfined can transition to other domains.
+
+Signed-off-by: Chris PeBenito <Christopher.PeBenito@microsoft.com>
+---
+ policy/modules/kernel/domain.te | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+MSFT_TAG: needs investigation
```
[the remark this answers is not in the document]
Comment (pebenito) [reply SPECS/selinux-policy/0033-domain-Unconfined-can-transition-to-other-domains.patch:11]: I need to understand the situation better to determine if the patch is upstreamable.

**Pull request merged**